src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length);
+
+    public String getRandomColor() {
+        return Colors.values()[index].toString();
+    }
```

Comment:
You have one index all the time.
getRandomColor returns the same color all time.